packit/cli/builds/in_image_builder.py

```diff
@@ -0,0 +1,88 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+import time
+from os import getcwd
+
+import click
+
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception, get_packit_api
+from packit.config import pass_config
+
+logger = logging.getLogger("packit")
+
+
+@click.command("in-image-builder")
+@click.option(
+    "--image-type",
+    default="aws",
+    type=click.STRING,
+)
+@click.option(
+    "--image-architecture",
+    default="x86_64",
+    type=click.STRING,
+)
+@click.option("--nowait", is_flag=True, default=False, help="Don't wait on build")
+@click.argument("path_or_url", type=LocalProjectParameter(), default=getcwd())
+@click.argument("image_name", default=None, type=click.STRING)
+@click.argument("image_distribution", default=None, type=click.STRING)
+@click.argument("image_account_id", default=None, type=click.STRING)
+@click.argument("packages_to_install", default=None, type=click.STRING)
+@click.argument("copr_namespace", default=None, type=click.STRING)
+@click.argument("copr_project", default=None, type=click.STRING)
+@click.argument("copr_chroot", default=None, type=click.STRING)
```

Comment:
Could this be cleaned up by using Copr build ID?

Reply:
Yeah, probably it could. I'm not sure what exactly we'll be passing from worker.

I like that the build ID is only a single value and not three.

Reply:
implemented